src/fparser/two/tests/test_Fortran2003.py

```diff
@@ -66,18 +66,17 @@
 from fparser.two.Fortran2003 import *
 from fparser.api import get_reader
 
-from nose.tools import assert_equal
 import pytest
 
 from fparser.two.parser import ParserFactory
 # this is required to setup the fortran2003 classes
 _ = ParserFactory().create(std="f2003")
 
 
-def assertRaises(exc, cls, s):
+def assertRaises(exc, cls, string):
     try:
-        cls(s)
-        raise AssertionError('Expected %s but got nothing' % exc)
+        cls(string)
+        raise AssertionError('Expected %string but got nothing' % exc)
```

Comment:
I'm not sure that this is right? Maybe use the newer, `"".format()` form instead of `%`?

Reply:
It wasn't, thank you for pointing it out. It is corrected now. I also added function description (from what I've gathered it's supposed to be doing) and changed the function name to conform PEP8 style guide.

Reply:
P.P.S. I thought of creating a configuration file for F2003 tests (similar to recently added F2008) but I see that @rupertford has already done it in PR #100.